src/Gimbal/GimbalController.cc

```diff
@@ -611,35 +608,34 @@ void GimbalController::sendGimbalRate(float pitch_rate_deg_s, float yaw_rate_deg
     }
 }
 
-/// Build and send MAV_CMD_DO_GIMBAL_MANAGER_PITCHYAW with rate fields,
-/// using Vehicle::sendMessageOnLinkThreadSafe
 void GimbalController::_sendGimbalRateCommandLong(float pitch_rate_deg_s,
-                                                  float yaw_rate_deg_s,
-                                                  uint32_t flags)
+                                                  float yaw_rate_deg_s)
 {
     auto sharedLink = _vehicle->vehicleLinkManager()->primaryLink().lock();
     if (!sharedLink) {
         qCDebug(GimbalControllerLog) << "_sendGimbalRateCommandLong: primary link gone!";
         return;
     }
 
+    uint32_t flags = 0;
```

Comment:
You should set `GIMBAL_MANAGER_FLAGS_ROLL_LOCK | GIMBAL_MANAGER_FLAGS_PITCH_LOCK`.

If you want to lock to some angle relative to North, you would also set `GIMBAL_MANAGER_FLAGS_YAW_LOCK`, otherwise it's relative to vehicle forward and will turn with the vehicle.